runtime/lua/vim/_meta/api.lua

```diff
@@ -321,13 +325,20 @@ function vim.api.nvim_buf_del_extmark(buffer, ns_id, id) end
 
 --- Unmaps a buffer-local `mapping` for the given mode.
 ---
+--- @see `nvim_del_keymap()`
```

Comment:
Some has backticks, some does not, as a result of mixed use of `@see foo` and `@see |bar|`.

Could be out of scope of this PR, but I'd like to note that LuaCATs (https://luals.github.io/wiki/annotations/#see) does not suggest the referenced symbol be quoted; without backticks LSPs these can be "hovered" whereas inline code blocks can't.

Reply:
> mixed use of `/// @see foo` and `/// @see |bar|` in the source file.

Which source files? all the `_meta/*` files should consistently use backticks. I didn't find any `|` in _meta files.

> LuaCATs does not suggest the referenced symbol be quoted; without backticks LSPs these can be "hovered"

Definitely worth considering. @lewis6991 what do you think?

Reply:
Sorry it wasn't clear enough; I meant `src/api/*.c` files. e.g.:
- `/// @see |nvim_set_keymap()|` (api/buffer.c)
- `/// @see nvim_create_user_command` (api/command.c)

In this PR's diff, those who don't have backticks include: `nvim_create_user_command`, `nvim_err_write`, `feedkeys`, `nvim_get_hl_by_name`, `nvim_get_hl_by_id`, `replace_termcodes`, includes `cpoptions`.

Reply:
The use of `|x|` is intentional for things that have `:help` tags. Those docs get rendered as :help and we want those tags to work there.

Hard to automate that without introducing false positives.